internal/flag/flags.go

```diff
@@ -15,241 +15,252 @@
+	SSEEnabled                               = "sseEnabled"                      // SSEEnabled flag
+	PathStyleAccessEnabled                   = "pathStyleAccessEnabled"          // PathStyleAccessEnabled flag
+	ReferenceTimeZoneOffset                  = "referenceTimeZoneOffset"         // ReferenceTimeZoneOffset flag
+	DailySnapshotRetentionDays               = "dailySnapshotRetentionDays"      // DailySnapshotRetentionDays flag
+	ClusterCheckpointIntervalMin             = "clusterCheckpointIntervalMin"    // ClusterCheckpointIntervalMin flag
+	SnapshotIntervalHours                    = "snapshotIntervalHours"           // SnapshotIntervalHours flag
+	SnapshotRetentionDays                    = "snapshotRetentionDays"           // SnapshotRetentionDays flag
+	WeeklySnapshotRetentionWeeks             = "weeklySnapshotRetentionWeeks"    // WeeklySnapshotRetentionWeeks flag
+	PointInTimeWindowHours                   = "pointInTimeWindowHours"          // PointInTimeWindowHours flag
+	ReferenceHourOfDay                       = "referenceHourOfDay"              // ReferenceHourOfDay flag
+	ReferenceMinuteOfHour                    = "referenceMinuteOfHour"           // ReferenceMinuteOfHour flag
+	MonthlySnapshotRetentionMonths           = "monthlySnapshotRetentionMonths"  // MonthlySnapshotRetentionMonths flag
+	Policy                                   = "policy"                          // Policy flag
+	SystemID                                 = "systemId"                        // SystemID flag
+	Default                                  = "default"                         // Default flag
+	Timestamp                                = "timestamp"                       // Timestamp flag
+	LiveMigrationDestinationClusterName      = "destinationClusterName"          // LiveMigrationDestinationClusterName flag
+	LiveMigrationDestinationProjectID        = "destinationProjectID"            // LiveMigrationDestinationProjectID flag
+	LiveMigrationHost                        = "migrationHost"                   // LiveMigrationHost flag
+	LiveMigrationSourceClusterName           = "sourceClusterName"               // LiveMigrationSourceClusterName flag
+	LiveMigrationSourceProjectID             = "sourceProjectID"                 // LiveMigrationSourceProjectID flag
+	LiveMigrationSourceSSL                   = "sourceSsl"                       // LiveMigrationSourceSSL flag
+	LiveMigrationSourceCACertificatePath     = "sourceCACertificatePath"         // LiveMigrationSourceCACertificatePath flag
+	LiveMigrationSourceManagedAuthentication = "sourceManagedAuthentication"     // LiveMigrationSourceManagedAuthentication flag
+	LiveMigrationSourceUsername              = "sourceUsername"                  // LiveMigrationSourceUsername flag
+	LiveMigrationSourcePassword              = "sourcePassword"                  // LiveMigrationSourcePassword flag
+	LiveMigrationDropCollections             = "drop"                            // LiveMigrationDropCollections flag
```

Comment:
These are the new flags, the new tab indentation has marked up everything else here.